components/python/curio/Makefile

```diff
@@ -0,0 +1,37 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# This file was automatically generated using the following command:
+#   $WS_TOOLS/python-integrate-project curio
+#
+
+BUILD_STYLE = setup.py
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =		curio
+COMPONENT_VERSION =		1.6
+COMPONENT_SUMMARY =		curio - Curio
+COMPONENT_PROJECT_URL =		https://github.com/dabeaz/curio
```

Comment:
The summary is nonsense.

Reply:
Yes, it is, but it is what curio's authors wants to see as a description for their project: see https://pypi.org/project/curio/ and https://github.com/dabeaz/curio/blob/master/setup.py#L15

But maybe it is worth changing...

Reply:
Fixed